version/version.go

```diff
@@ -10,3 +10,11 @@ var (
 	// Revision is filled with the VCS revision. Filled in at linking time.
 	Revision = ""
 )
+
+func GetVersion() string {
```

Comment:
Wonder if it's necessary to have a getter here since `Version` is already exported. And by the Golang convention, `Getter` can be renamed to `Version()`. https://go.dev/doc/effective_go#Getters

Reply:
We decided to add the getters like that in a group discussion.

Reply:
Gotcha